### ISSUE TITLE
Access Bug

### ISSUE THREAD

**Issue by gep5** [Markemp/Cryengine-Converter#26]
I think I've installed everything correctly and am using this command to bulk convert .skin files.

cgf-converter *.skin -objectdir (Directory Containg .skin files)

Here is the bug I get

https://imgur.com/a/ttqt1nA

And the files I'm trying to convert (Contains non .skin files).

https://mega.nz/#!l4BSVYwY!gc9nUjs2UG3kECFsfrxpcDCzXkAJJYSR3oQAhLdM4zc

Thank you :)

**Comment by gep5** [comment]
The game I'm porting from (Ghost Sniper 3) also uses .cdf files for its models.

Does the tool support those at the current time?

**Comment by gep5** [comment]
Quick update, I also got a second bug regarding something about an out of bounds index.

https://imgur.com/a/VbE8ZVN

**Comment by Markemp** [comment]
Will take a look at this soon.

**Comment by gep5** [comment]
Thanks for taking a look at this. If it helps, I believe for this game the armatures are stored in the .chr files.

**Comment by Markemp** [comment]
https://gyazo.com/8ae63e589d31997674a18a84c33dbae6

Been looking at this and it does look like the converter can handle Ghost Sniper 3 models.  I believe the issue you're having is with the way the command is being run.  You're running it in c:\windows\system32, but it needs to be put somewhere more functional.  For my setup, I have it put into a Scripts directory on my D: drive, so you can find cgf-converter.exe in D:\scripts.  I also have that directory in the path, so I can access it from any other path without having to put the full name (D:\scripts\cgf-converter.exe) every time I want to use it.

Navigate to the directory with the skin files you are trying to convert, and then run the command.  That should work a lot better.  Let me know if you are still having problems.

You may also want to watch the tutorial videos at https://www.youtube.com/playlist?list=PL106ZeLhxxVn551_IKGKeU_LBODtkh29b, as it goes over how to set everything up as well.

**Comment by gep5** [comment]
Sorry for such a delayed response here, I had system issues and had to do a reinstall. I've got SWG3 reinstalled and unpacked now and I'll give this a go again over the coming days.

Cheers :)

**Comment by gep5** [comment]
Pleased to report that everything works as intended now. Thank you :)

By the way, is there a way to convert skeletons as well? The meshes don't have any weight painting I know, but I could probably do that bit myself using the skeleton as a guide.

**Comment by Markemp** [comment]
Skeletons should convert as well, although those are usually found in .chr files.  This does include weight painting as well.

**Comment by gep5** [comment]
> Skeletons should convert as well, although those are usually found in .chr files. This does include weight painting as well.

The skeleton file does convert, but for some reason, instead of bones I just get a single polygon. Is there something about these specific files throwing the convertor off? I've included both the original .chr skeleton and the .dae conversion.

https://we.tl/t-q2kb8HkSX7

I converted these files by placing cgf-convertor.exe in the same directory as the files and dropping them ontop the .exe.
